packages/wonder-blocks-icon-button/components/icon-button-core.js

```diff
@@ -0,0 +1,143 @@
+// @flow
+import React from "react";
+import {StyleSheet} from "aphrodite";
+
+import Color, {mix, fade} from "wonder-blocks-color";
+import {addStyle} from "wonder-blocks-core";
+import type {ClickableHandlers} from "wonder-blocks-core";
+import type {SharedProps} from "./icon-button.js";
+
+type Props = SharedProps &
+    ClickableHandlers & {
+        hovered: boolean,
+        focused: boolean,
+        pressed: boolean,
+    };
+
+const StyledButton = addStyle("button");
+
+export default class IconButtonCore extends React.Component<Props> {
+    render() {
+        const {
+            icon,
+            alt,
+            color,
+            kind,
+            light,
+            disabled,
+            testId,
+            style,
+            hovered,
+            focused,
+            pressed,
```

Comment:
We should have `href` and `clientSideNav` props similar to `Button`.  Sorry that I left this out the IconButton spec doc.  I'll update it but basically copy the same behavior as `Button` for those two props.

Reply:
Will `onClick` be optional then?